packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/BotMessage.tsx

```diff
@@ -127,19 +127,36 @@ _Italic text, formatted with single underscores_
 
   return (
     <>
-      <Message name="Bot" role="bot" avatar={patternflyAvatar} content={`Text-based message from a bot named "Bot"`} />
-      <Message role="bot" avatar={patternflyAvatar} content="Text-based message from a nameless bot" />
+      <Message
+        name="Bot"
+        role="bot"
+        avatar={patternflyAvatar}
+        content={`This is a text-based message from a bot named "Bot."`}
+      />
+      <Message
+        name="Bot"
+        role="bot"
+        avatar={patternflyAvatar}
+        content={`This is a text-based message from "Bot," with an updated timestamp.`}
+        timestamp="1 hour ago"
+      />
+      <Message name="Bot" role="bot" avatar={patternflyAvatar} content={code} />
+      <Message name="Bot" role="bot" avatar={patternflyAvatar} content={orderedList} />
+      <Message name="Bot" role="bot" avatar={patternflyAvatar} content={unorderedList} />
+      <Message name="Bot" role="bot" avatar={patternflyAvatar} content={moreComplexList} />
+      <Message name="Bot" role="bot" avatar={patternflyAvatar} content="Example content" isLoading />
```

Comment:
We can remove lines 143 through 146 since the tables PR merged - the other examples should be further down now.